scopes/component/merging/merge-cmd.ts

```diff
@@ -22,18 +22,18 @@ export class MergeCmd implements Command {
   extendedDescription = `merge changes of the remote head into local. when on a lane, merge the remote head of the lane into the local.
 if no ids are specified, all pending-merge components will be merged. (run "bit status" to list them).
 optionally use '--abort' to revert the last merge. to revert a lane merge, use "bit lane merge-abort" command.
-${WILDCARD_HELP('merge')}`;
+${WILDCARD_HELP('merge')}`; // @david does this command auto-snap by default? If so we must document it here
   alias = '';
   options = [
-    ['', 'ours', 'in case of a conflict, override the used version with the current modification'],
-    ['', 'theirs', 'in case of a conflict, override the current modification with the specified version'],
-    ['', 'manual', 'in case of a conflict, leave the files with a conflict state to resolve them manually later'],
-    ['', 'abort', 'in case of an unresolved merge, revert to the state before the merge began'],
+    ['', 'ours', 'in case of a conflict, keep the local modification'],
+    ['', 'theirs', 'in case of a conflict, override the local modification with the specified version'],
+    ['', 'manual', 'in case of a conflict, leave files in conflict state to resolve manually later'],
+    ['', 'abort', 'in case of an unresolved merge, revert to pre-merge state'],
     ['', 'resolve', 'mark an unresolved merge as resolved and create a new snap with the changes'],
-    ['', 'no-snap', 'do not auto snap in case the merge completed without conflicts'],
+    ['', 'no-snap', 'do not auto snap even if the merge completed without conflicts'], // @david please review this change
```

Comment:
It's not accurate. There can be different scenarios:
1. there are conflicts in the files - we don't snap. 
2. it's possible to merge without snap - we don't snap. this is when local is ahead or the remote is ahead. 
3. if local and remote are diverged, we need a new snap, which will have two parents: one is the local and one is the remote. That's how we save both history.

Reply:
hmm ok
i'm just not sure how that conflicts with the change i made
or more, "in case" also doesnt seem to describe what you're saying

Should it be "when merge completed without conflicts"? Or something else?

Reply:
sorry, it's actually good your change. 
I'm wondering whether we should add the "diverge" here. Maybe 
"do not auto snap in case of diverge even if the merge completed without conflicts". 
but not sure how easy it it to explain this "diverge" in this context.. so up to you..

Reply:
do you mean adding to the general description? i.e. that even without the flag, bit wont auto-tag if there's a diverge, even when the merge was successful?

Reply:
Yes actually it can be part of the extended description. 
I want to clarify that all this "auto-snap" we do is only in case the local and remote have diverged. Because that's the only case we need to make a new snap that has two parents: one from the local and one from the remote.

In other scenarios, e.g. the local is ahead or the remote is ahead, there is no need for this snap.

Reply:
ok great, i'll try to word that in the extended description